CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Karafka framework changelog
 
+## 2.1.0 (Unreleased)
```

Comment:
we don't have to release as 2.1.0 but as 2.0.34 if we reference for now with the old name as an alias.